priv/riak_shell/riak_shell_regression1.log

```diff
@@ -78,8 +79,14 @@ family1,series1,1970-01-01T00:00:00.007Z,cloudy,27.9
 {{command, "INSERT INTO GeoCheckin2 (region, state, time, weather, temperature) VALUES ('South Atlantic', 'South Carolina', 1452252523189, 'snowing', 19.2);\n"}, {result, "Inserted 1 row."}}.
 {{command, "SELECT 555, 1.1, 1e1, 1.123e-2 from GeoCheckin2 WHERE time > 1452252523182 AND time < 1452252543182 AND region = 'South Atlantic' AND state = 'South Carolina';\n"}, {result, "555,1.1,10.0,0.01123
 555,1.1,10.0,0.01123
-555,1.1,10.0,0.01123
```

Comment:
This line should not have been removed. There should be 3 rows returned. Perhaps it's a bit confusing b/c of the intentionally hardcoded values in the SELECT clause, but this is testing A) the number of rows returned; B) that values of each of the types in the SELECT clause are properly returned.